child_compassion/wizards/global_child_search.py

```diff
@@ -38,6 +38,8 @@ class GlobalChildSearch(models.TransientModel):
         string="Field Offices",
         readonly=False,
     )
+    is_exhaustive = fields.Boolean(string="Make exhaustive search",
+                                   help="Get a bit of all children depending on the search criteria")
```

Comment:
This was another attempt and can be removed, right? It looks like it's never used.